rust/xaynet-server/src/state_machine/phases/sum2.rs

```diff
@@ -181,11 +181,8 @@ mod test {
             updater.compose_update_message(coord_keys.public, &sum_dict, scalar, model.clone());
         let masked_model = utils::masked_model(&msg);
         let local_seed_dict = utils::local_seed_dict(&msg);
-        let mut aggregation = Aggregation::new(
-            utils::mask_settings().into(),
-            utils::mask_settings().into(),
-            model_size,
-        );
+        let config: MaskConfig = utils::mask_settings().into(); // help out the typechecker
```

Comment:
i'd say this extra comment is not necessary (because kinda obvious), but of course also not harmful.
```suggestion
        let config: MaskConfig = utils::mask_settings().into();
```